src/foam/nanos/crunch/CapabilityRuntimeException_disabled.txt

```diff
@@ -6,13 +6,16 @@
 package foam.nanos.crunch;
 
 import java.util.ArrayList;
+import foam.nanos.crunch.lite.Capable;
 import foam.nanos.auth.AuthorizationException;
 
-public class CapabilityRuntimeException extends AuthorizationException {
-  private ArrayList<String> capabilities;
+public class CapabilityRuntimeException extends AuthorizationException implements foam.core.NetworkException {
```

Comment:
Can this class be removed now?

Reply:
CapabilityRuntimeException is still used. NetworkException was made because default properties can't be added to interface (I wanted to change `foam.core.Exception` itself but couldn't do so without breaking other implementations).

Now that you mention it, since exceptions are modelled now I think we can combine three models into one:
```
from
CapabilityRuntimeException + CapabilitiesRequiredRemoteException + CapabiliityIntercept
to
CapabilityIntercept
```

which would be more specifically `CapabilityIntercept extends RemoteException javaExtends AuthorizationException implements foam.core.NetworkException`